code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -366,8 +366,8 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	new/datum/stack_recipe("sender - lever", /obj/structure/destructible/clockwork/trap/trigger/lever, 1, time = 10, one_per_turf = TRUE, on_floor = TRUE), \
 	new/datum/stack_recipe("sender - repeater", /obj/structure/destructible/clockwork/trap/trigger/repeater, 2, time = 20, one_per_turf = TRUE, on_floor = TRUE), \
 	null,
-	new/datum/stack_recipe("receiver - brass skewer", /obj/structure/destructible/clockwork/trap/brass_skewer, 2, time = 20, one_per_turf = TRUE, on_floor = TRUE), \
-	new/datum/stack_recipe("receiver - steam vent", /obj/structure/destructible/clockwork/trap/steam_vent, 3, time = 30, one_per_turf = TRUE, on_floor = TRUE), \
+	new/datum/stack_recipe("receiver - brass skewer", /obj/structure/destructible/clockwork/trap/brass_skewer, 2, time = 20, one_per_turf = TRUE, on_floor = TRUE, adjacency_checks = TRUE), \
```

Comment:
Could you make these into flags ?  Something like `placement_requirements = ADJACENT / placement_requirements = ADJACENT_CARDINAL` That way the system would be more flexible in the future.

Reply:
I'm not sure how to do flags!

Reply:
Same thing you do just instead of n variables you have one common one.

Reply:
Err, did you mean bitflags, or just define checks? I should have clarified that it's *bitflags* I'm no good at, but the other method I can do easily.